index.html

```diff
@@ -10,31 +10,57 @@
+              <li><a href="#"> <i class="fa fa-apple"></i></a></li>     
               <li><a href="#">Store</a></li>
               <li><a href="#">Mac</a></li>
               <li><a href="#">iPod</a></li>
               <li><a href="#">iPhone</a></li>
               <li><a href="#">iPad</a></li>
               <li><a href="#">iTunes</a></li>
               <li><a href="#">Support</a></li>
-              <li><a href="#">Support</a></li>
-              <li><input type="text" name="sear-box"><i class="fa fa-search"></i></li>
+              <li class="last-item"><input type="text" name="sear-box"><i class="fa fa-search"></i></li>
            </ul>
         </nav>
     </header>
     <main>
-
-
+    <section class="call-to-action"> 
+        <div class="page-highlight">
+            <h2>Everyone has something to share. See how people are using<br>
+             iPad in their unique ways to add to the world's story.
+            </h2>
+          <a href="" >Watch the "Your verse" film and explore the stories
+          <i class="fa fa-play-circle"></i></a>
+       </div>    
+    </section>     
+    <section class="product-sample">
+        <h2 class="product-types">Sample products</h2>
+              <ul>
+                <li><a href="#"><img src="https://web.archive.org/web/20140228164946im_/http://images.apple.com/home/images/promo_iphone5s.jpg" alt="iPhone 5s"></a></li>
+                <li><a href="#"><img src="https://web.archive.org/web/20140228164946im_/http://images.apple.com/home/images/promo_iphone5c.jpg" alt="iPhone 5c"></a></li>
+                <li><a href="#"><img src="https://web.archive.org/web/20140228164946im_/http://images.apple.com/home/images/promo_30_years.jpg" alt="happy birthday mac"></a></li>
+                <li><a href="#"><img src="https://web.archive.org/web/20140228164946im_/http://images.apple.com/home/images/promo_supplier_responsibility.jpg" alt="supplier responsibility"></a></li>
+              </ul>
+    </section>  
     </main>
-    <aside>
-
-    </aside>
     <footer>
-        
+        <div>
+           <p>Shop the <a href="#">Apple Online Store</a> (1-800-MY-APPLE), visit an <a href="#">Apple Retail Store,</a> or find a <a href="#">reseller.</a></p><br>
+           <p>&copy; 2020 Apple Inc. All rights reserved. <span><a href="#">Terms of Use</a></span><a href="#">Privacy Policy</a></p>
+        </div>
+        <nav>
+            <ul class="footer-nav">
+                <li><a href="#">Site Map</a></li>
+                <li><a href="#">Hot News</a></li>
+                <li><a href="#">RSS Feeds</a></li>
+                <li><a href="#">Media Info</a></li>
+                <li><a href="#">Environment</a></li>
+                <li><a href="#">Job Opportunities</a></li>
+                <li><a href="#">Contact Us</a></li>
+                <li><a href="#"><img src="images/usa-flag.jpeg" alt="USA Flag"></a></li>
+            </ul>
+        </nav>
```

Comment:
- [ ] Check this [link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/nav) to see the proper use of the `nav`  tag (see Usage section). You shouldn't use it in the footer.